desktop_app/loading/loading.css

```diff
@@ -0,0 +1,547 @@
+@charset "UTF-8";
```

Comment:
Before merging let's do a couple of things:
1. move everything from `desktop_app` to another folder called simply `desktop` or `electron`.
2 Let's try to change the **installation.png** to something more modern or, at least, more Netbeast-branded.